guillotina_elasticsearch/schema.py

```diff
@@ -70,6 +70,14 @@ def get_mappings(schemas=None, schema_info=False):
                     field_mapping['_schemas'] = []
                 if schema.__identifier__ not in field_mapping['_schemas']:
                     field_mapping['_schemas'].append(schema.__identifier__)
+
+            if index_name in mappings and mappings[index_name] != field_mapping:
```

Comment:
field_mapping is a dict ? In that case that are equal just mean its the same dict

Reply:
Yes, `field_mapping` is a dict. 

> I whould prefer a warning and not an exception, as it may be a use case that you want to map two fields on the same index on different content type.

I don't get your point. This is only failing if two fields on the same index defines different mappings. There is no problem on defining same mapping on same field in different contents.

I added another test to clarify the behavior